dev-test/githunt/types.reactApollo.tsx

```diff
@@ -224,7 +224,7 @@ export type VoteMutation = ({ __typename?: 'Mutation' } & { vote: Maybe<({ __typ
 
 import gql from 'graphql-tag';
 import * as React from 'react';
-import * as ReactApollo from 'react-apollo';
```

Comment:
@ardatan it's treated as a namespace, so it won't increase bundle size, totally safe.

Reply:
https://github.com/kamilkisiela/tree-shake-test/tree/apollo

Here's a playground